src/sim_init.cc

```diff
@@ -160,6 +160,10 @@ void SimInit::LoadInfo() {
   // should be updated to uint64_t.
   si_.dt = qr.GetVal<int>("DurationSecs");
 
+  qr = b_->Query("RNGInfo", NULL);
+  si_.seed = qr.GetVal<int>("Seed");
+  si_.stride = qr.GetVal<int>("Stride");
+
```

Comment:
Note above comment about where the data is added - it's in two places above and I advise one that is different from this one.